gnomad/utils/filtering.py

```diff
@@ -436,6 +439,8 @@ def filter_to_gencode_cds(
     :param gencode_ht: Gencode Table to use for filtering the input Table/MatrixTable
         to CDS regions. Default is None, which will use the default version of the
         Gencode Table resource.
+    :param genes: Gene(s) to filter to. Default is None, which will filter to all genes.
```

Comment:
Might want to support either gene symbol or gene id? Either way, specify which one is expected